tools/deployment/make_osx_package.sh

```diff
@@ -50,19 +52,24 @@ set -e
 "
 
 POSTINSTALL_AUTOSTART_TEXT="
-if launchctl list | grep -qcm1 osquery; then
+if launchctl list | grep -qcm1 $LD_IDENTIFIER; then
   launchctl unload $LD_INSTALL
 fi
 cp $LAUNCHD_DST $LD_INSTALL
 launchctl load $LD_INSTALL
 "
 
+POSTINSTALL_CLEAN_TEXT="
+rm -rf $OSQUERY_DB_LOCATION
```

Comment:
No objections to this? It's a little nasty, but seems to work fine. I definitely think that being to start the daemon fresh is a useful feature.